src/npr/ont.py

```diff
@@ -105,27 +113,42 @@ def sleep():
                 fnames.sort(key=os.path.getctime)
                 n = int(fnames[-1].split("-")[-1].split(".")[0]) + 1  # get new run number
             
+            # use conda prefix if specified
+            conda_prefix = config["snakemake"]["conda_prefix"]
+            conda_prefix = conda_prefix if conda_prefix else None
+
+            print("Starting snakemake on file {} with configfile {} using workdir {}..."
+                  .format(snakefile_file, configFile, output_directory), file=sys.stderr)
             snak_stat = snakemake.snakemake(
                 snakefile = snakefile_file,
-                debug = True,
-                #cores = 25,
+                #debug = True,
+                cores = config["snakemake"]["cores"],
```

Comment:
One option here might be to simply pass **config['snakemake'] to snakemake.snakemake. This way, any arbitrary snakemake options can be passed via the config.